builtin/settingtypes.txt

```diff
@@ -626,8 +626,8 @@ shadow_map_color (Colored shadows) bool false
 #    Spread a complete update of shadow map over given amount of frames.
 #    Higher values might make shadows laggy, lower values
 #    will consume more resources.
-#    Minimum value: 1; maximum value: 16
-shadow_update_frames (Map shadows update frames) int 8 1 16
+#    Minimum value: 1; maximum value: 8
+shadow_update_frames (Map shadows update frames) int 1 1 8
```

Comment:
1 has quite a performance difference on my machine (with Intel or NVidia cards).
With #12364 we can leave this as is I think. (both default and max value)

Reply:
Done